CHANGELOG.md

```diff
@@ -1,50 +1,5 @@
 # Release History
 
-## 1.0.0-beta.4 (upcoming)
+[Communicatin UI Library CHANGELOG](docs/CHANGELOG_UI.md)
```

Comment:
Spelling syntax error, this one and the one just below

Reply:
Thanks, fixed